ApollosProject/apollos-prototype
# Cleans up front-end fragments and removes LikeIcon from FeedView

## DESCRIPTION

### What does this PR do, or why is it needed?

Changes you will see in this PR

1. Removes a ton of fields that were queried for before they were used, mostly in the feed view
- Fields like Audio, `ContentSeries` on the `UniversalContentItem` (no label is rendered) 
2. Removes `isLiked` from the content feeds. 
3. Removed the `largeCardFragment` from the `ContentCard`. It wasn't any different than the base, and we don't use it anymore. 
4. Removes `isLive` from the `baseCardFragment`. A different PR is augmenting how this data will make it's way to the card tile. By removing `isLive` along with `isLiked`, this query will be 100% cacheable. 

### What design trade-offs/decisions were made?

n/a

### How do I test this PR?

1. Open the app.
2. Like an item, make sure it shows up on your profile feed
3. Unlike an item, make sure it shows up on your profile feed. 
4. Tap around and make sure that `ContentItems` render and `open` as expected. Shouldn't be any changes besides the Like icon disappearing from the feed view.

![2019-11-01 11 12 42](https://user-images.githubusercontent.com/1637694/68034698-90700600-fc98-11e9-9690-99c4e0c189e9.gif)


### What automated tests did you write?

Updated existing tests.

## TODO:


## REVIEW:

**Manual QA**


**Code Review: Questions to consider**


> The purpose of PR Review is to _improve the quality of the software._
